libtransmission/cache.cc

```diff
@@ -203,16 +210,16 @@ int Cache::prefetch_block(tr_torrent* torrent, tr_block_info::Location const& lo
 
 int Cache::flush_span(CIter const begin, CIter const end)
 {
-    for (auto walk = begin; walk < end;)
+    for (auto span_begin = begin; span_begin < end;)
     {
-        auto const [contig_begin, contig_end] = find_contiguous(begin, end, walk);
+        auto const span_end = find_span_end(span_begin, end);
```

Comment:
So `Cache::flush_span()` no longer search backwards for a new `span_begin`?

To ba fair, with the current usage, it doesn't need to.

Reply:
Correct; no longer needed